android-quick-start/README.md

```diff
@@ -0,0 +1,88 @@
+# Quickstart: Create an Android app with Amazon Location Maps and Places API
+
+This quickstart app creates a quick start application that demonstrates the following:
+
+1. Adding Amazon Location Maps using Esri as data source to the app.
+2. Adding reverse geocoding using Amazon Location Places API to demonstrate a place picker.
+
+After a successful build the app will look like:
+
+![](screenshot.png)
+
+## Prerequisities
+1. A Amazon Location map resource
+1. A Amazon Location place-index resource
+1. A Cognito identity pool for unauthorized access
+
+## How to set up the project
+Before the project can be built we need the following steps to be completed:
+1. Configuring amplify to set map, place-index, and cognito identity-pool id.
```

Comment:
1. Configuring Amplify to set map, place-index, and Cognito Identity Pool id.

Reply:
I made this change